docs/usage/migration.rst

```diff
@@ -0,0 +1,31 @@
+..
+    This file is part of Invenio.
+    Copyright (C) 2015-2019 CERN.
+
+    Invenio is free software; you can redistribute it and/or modify it
+    under the terms of the MIT License; see LICENSE file for more details.
+
+
+[WIP] Data Migration
+--------------------
+
+:code:`Locations` are used to represent different storage systems and possibly
+different geographical locations. :code:`Buckets` but also :code:`Objects` are
```

Comment:
I am not convinced by `geographical`, but it might make sense and I miss some history when this was created. We might ask Lars confirmation here.

Reply:
Its not like we came up with it, I saw it here https://github.com/inveniosoftware/invenio-files-rest/blob/4ba19081e738bac8c00e0ddbbc95499ab54ba009/invenio_files_rest/models.py#L26